doxygen/microservices.cpp

```diff
@@ -103,6 +103,7 @@
   - #msiDeleteCollByAdmin  - Deletes a collection by administrator
   - #msiRenameLocalZone  - Renames the local zone by updating various tables
```

Comment:
does this need to be updated/removed?

Reply:
See `acRenameLocalZone` in the core.re.template.

Administrators will need to manually update their core.re files for this to work.

Reply:
Okay - but new installs (where nearly all zone renaming happens...) would have the new core.re...

Do we need to update the implementation in the default_cpp rule engine too?

Reply:
I believe that change is in the PR

Reply:
"I see" said the blind man.